ui/src/app/cluster-workflow-templates/cluster-workflow-template-list.tsx

```diff
@@ -76,17 +76,9 @@ export function ClusterWorkflowTemplateList({history, location}: RouteComponentP
                         <div className='columns small-5'>NAME</div>
                         <div className='columns small-3'>CREATED</div>
                     </div>
-                    {templates.map(t => (
-                        <Link className='row argo-table-list__row' key={t.metadata.uid} to={uiUrl(`cluster-workflow-templates/${t.metadata.name}`)}>
-                            <div className='columns small-1'>
-                                <i className='fa fa-clone' />
-                            </div>
-                            <div className='columns small-5'>{t.metadata.name}</div>
-                            <div className='columns small-3'>
-                                <Timestamp date={t.metadata.creationTimestamp} />
-                            </div>
-                        </Link>
-                    ))}
+                    {templates.map(t => {
+                        return <ClusterWorkflowTemplateRow workflow={t} key={`${t.metadata.namespace}/${t.metadata.name}`} />;
+                    })}
```

Comment:
Conversely, I don't think this row needs to be split into a separate component at this time. It can be left as-is within the list component. The row is quite short here (note that the Workflows Row has _much_ more details than this row)

Reply:
It would be easier to renovate the system in the future if it were structured in the same way.
Personally, I would like to keep it the same. Is that a problem?

Reply:
Ah I see what you were going for, because `workflows-row` is split it out, make the others consistent.

Two things:
1. The Workflows List is the exception to things because it's so big and detailed; I'm not sure that other components should follow its example
1. If we do want to refactor this as such, it would be better done as a separate refactor PR. That keeps things "atomic" and single purpose -- in particular, it keeps the diff here to only what is necessary for this feature, which makes it easier to review now and easier to understand in the future if someone looks at this PR (if this needs a revert for some reason, for historical purposes, or if someone want to copy this PR to another feature, etc). Similarly a refactor is easier to read / review / revert etc as its own independent PR -- the changes should effectively be a no-op as they do not change behavior.

Reply:
Also I see that the variables here are a bit harder to use without a separated component. That is a bit annoying.

There's a few techniques to workaround that. Two main ones I am thinking of are to call a function to do some of the logic and to split out the title/desc as its own component, but in the same file for now. The latter would be more consistent and would be easier to follow-up with a refactor if desired

Reply:
Okay, I made the title and description separate components.